src/Compilers/CSharp/Portable/Symbols/FunctionPointers/FunctionPointerTypeSymbol.cs

```diff
@@ -26,7 +26,8 @@ public static FunctionPointerTypeSymbol CreateFromSource(FunctionPointerTypeSynt
                     suppressUseSiteDiagnostics));
 
         /// <summary>
-        /// Creates a function pointer from individual parts. This method should only be used when diagnostics are not needed.
+        /// Creates a function pointer from individual parts. This method should only be used when diagnostics are not needed. This is
+        /// intended for use in test code.
```

Comment:
Is there a way to enforce that? Consider at least renaming the method to "CreateFromPartsTestOnly"